inductiva/tasks/run_simulation.py

```diff
@@ -39,4 +45,43 @@ def run_simulation(
         raise RuntimeError(
             f"Expected result to be a string with task_id, got {type(task_id)}")
 
+    if format_utils.getenv_bool("SAVE_SIM_METADATA", True):
+        metadata = {
+            **{
+                "machine_group_id": resource_pool_id,
+                "task_id": task_id,
+                "input_dir": str(input_dir),
+                "storage_dir": str(storage_dir),
+            },
+            **kwargs,
+        }
+        if extra_sim_metadata_to_save is not None:
+            metadata = {**metadata, **extra_sim_metadata_to_save}
+        _save_metadata(save_sim_metadata_dir, metadata)
+
     return task
+
+
+def _save_metadata(save_sim_metadata_dir, metadata):
```

Comment:
let's always save this metadada to a file in the current working directory that only gets appended.
The ame of the file can be defined using a module specific global variable.